topics/ICRC-21/icrc_21_consent_msg.md

```diff
@@ -136,7 +136,13 @@ sequenceDiagram
 ```
 
 1. The relying party connects to the signer and requests a signature on a canister call.
-2. The signer fetches the consent message from the target canister:
+2. The chain-connected signer component fetches the consent message from the target canister:
+   * `icrc21_consent_message_request.method` must match the canister call method.
+   * `icrc21_consent_message_request.arg` must match the canister call argument.
+   * The hot signer component must use the anonymous identity for the `icrc21_consent_message` request.
+   * The `icrc21_consent_message` canister call must be made to the target canister.
```

Comment:
`.canisterId` is maybe missing?

Reply:
No, that's correct. `icrc21_consent_message_request.method` and `icrc21_consent_message_request.arg` refer to properties of the arguments of the  `icrc21_consent_message` canister call (of type `icrc21_consent_message_request`).

> The `icrc21_consent_message` canister call must be made to the target canister.

This makes sure that the canister id of the `icrc21_consent_message` and the requested canister call go to the same canister, which is a property independent of the arguments.

Reply:
I don't get it but, ok.